e2e/cypress/integration/api/iam/01_projects_api.spec.ts

```diff
@@ -0,0 +1,370 @@
+const admin_token = 'ZNhnqJyDqgkXVk_bpobS3YhIMz0='
```

Comment:
is it possible to instantiate and fetch this token? feels funny to have it hard-coded like this

Reply:
mmmm prob can't make one without one 🤔 conundrum

Reply:
I could but then I couldn't share the token value with all of the tests. Though I can try to figure something else out.